client/landing/gutenboarding/onboarding-block/index.ts

```diff
@@ -13,9 +13,9 @@ import edit from './edit';
 export const name = 'automattic/onboarding';
 
 export const settings: BlockConfiguration< Attributes > = {
-	title: NO__( 'Onboarding' ),
+	title: __( 'Onboarding' ),
 	category: 'layout', // @TODO
-	description: NO__( 'Onboarding wizard block' ),
+	description: __( 'Onboarding wizard block' ),
```

Comment:
I don't think we should translate these; let's remove the whole description instead.

Reply:
Might be required by block typings tho :grimacing:

Reply:
We can have empty strings, or "WordPress.com", but we don't need to have something to translate.

Reply:
144277448cf452b8feb4339166d2b703313b05c6